jujugui/static/gui/src/app/models/test-models.js

```diff
@@ -697,7 +706,7 @@ describe('test_model.js', function() {
 
       // XXX - We no longer use relation_errors but this test should remain
       // until it's completely removed from the codebase.
-      it.skip('should reset relation_errors',
+      xit('should reset relation_errors',
```

Comment:
Have we removed `relation_errors` from the code base yet? It looks like we just set it as an empty object but it otherwise doesn't exist in anything but tests. Would be nice to go ahead and finish removing it.

Not necessarily in this branch mind you, but given we're migrating a test for something we don't need, I thought it worth bringing up.

Reply:
I had a quick look last week and I think this will get removed when @frankban's jujulib work lands then we can remove this.